### ISSUE TITLE
[BUG] Some output from `/stats` in unreadable due to output colors not respecting used theme

### ISSUE THREAD

**Issue by MarcinOrlowski** [anthropics/claude-code#13523]
### Preflight Checklist

- [x] I have searched [existing issues](https://github.com/anthropics/claude-code/issues?q=is%3Aissue%20state%3Aopen%20label%3Abug) and this hasn't been reported yet
- [x] This is a single bug report (please file separate reports for different bugs)
- [x] I am using the latest version of Claude Code

### What's Wrong?

 Some output from `/stats` in unreadable due to used colors:

<img width="346" height="137" alt="Image" src="https://github.com/user-attachments/assets/70274f01-9cd0-4d34-aacb-60bb17a82f71" />

<img width="275" height="187" alt="Image" src="https://github.com/user-attachments/assets/99bf26a2-5449-457e-8c21-92b5a5e4b924" />

### What Should Happen?

All colors should be following used theme

### Error Messages/Logs

```shell

```

### Steps to Reproduce

Set theme to `Dark mode (colorblind-friendly)` and execute `/stats`

### Claude Model

None

### Is this a regression?

Yes, this worked in a previous version

### Claude Code Version

2.0.64

### Platform

Anthropic API

### Operating System

Ubuntu/Debian Linux

### Terminal/Shell

Other
